app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
```

Comment:
- [ ] **OPTIONAL**
Please consider using the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) `after_save` in order to call the `update_likes` method every time a new like has been saved, then make the method private